docs/source/getting-started/tabular-data/agg_and_filter.ipynb

```diff
@@ -0,0 +1,435 @@
+      "text/plain": [
+       "<Figure size 1200x600 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.figure(figsize=(12, 6))\n",
+    "sns.lineplot(x=count_year_all[\"Year-Quarter\"].to_list(), y=count_year_all['Actual Count'].to_list(), marker='o', label='Actual Count')\n",
+    "sns.lineplot(x=count_year_all[\"Year-Quarter\"].to_list(), y=count_year_all['DP Count'].to_list(), marker='o', label='DP Count')\n",
+    "\n",
+    "plt.title('Actual Count vs DP Count Over Years and Quarters')\n",
+    "plt.xlabel('Year-Quarter')\n",
+    "plt.ylabel('Count')\n",
+    "plt.xticks(rotation=45) \n",
+    "plt.legend()\n",
+    "\n",
+    "plt.grid(True)\n",
+    "plt.tight_layout()  \n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "While the differentially computed values still follow the general trend of the actual values, they do vary more significantly due to the smaller sizes of the year-quarter groups. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Filtering"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To demonstrate the `filter` method, compute the total number of respondents not working due to a layoff. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Actual Total Count of Laid Off Respondents:  76009\n",
+      "DP Total Count of Laid Off Respondents:  76115\n"
+     ]
+    }
+   ],
+   "source": [
+    "filt_actual = (df.filter(pl.col(\"ILOSTAT\")==3).select(pl.len())).collect().item()\n",
+    "print(\"Actual Total Count of Laid Off Respondents: \", filt_actual)\n",
+    "filt_dp = (context.query().filter(pl.col(\"ILOSTAT\")==3).select(pl.len().dp.noise())).release().collect().item()\n",
+    "print(\"DP Total Count of Laid Off Respondents: \", filt_dp)"
```

Comment:
Feel free to expand the filter section based on the conversation yesterday: Filtering can be seen as a kind of aggregation, and if the result of filtering is a small subset, you'll have a corresponding increase in noise and decrease in utility.